src/components/Book.js

```diff
@@ -1,27 +1,58 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import './Book.css';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   const onRemoveClicked = (id) => {
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h1>{book.title}</h1>
-      <h2>
-        Author:
-        {book.author}
-      </h2>
-      <button onClick={() => onRemoveClicked(book.item_id)} type="button">Delete</button>
+    <div className="main-book" key={book.item_id}>
+      <div>
+        <p className="category">{book.category}</p>
```

Comment:
I appreciate your dedication to aligning your project with the design. As previously mentioned, it's essential to ensure that your styling closely matches the design specifications. Specifically, please focus on details such as font size, font family, opacity, color, and margin. Make the necessary adjustments in your code to match the design as closely as possible.

By meticulously following the design guidelines, you'll create a user interface that is consistent with the intended visual aesthetics, providing a more polished and professional user experience. Keep up the great work, and thank you for your attention to detail! 🌟🎨